testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/multinode-gcs/iceberg.properties

```diff
@@ -0,0 +1,5 @@
+connector.name=iceberg
+hive.metastore.uri=thrift://hadoop-master:9083
+hive.config.resources=/docker/presto-product-tests/conf/presto/etc/hive-default-fs-site.xml
+iceberg.file-format=PARQUET
+hive.gcs.json-key-file-path=${ENV:GCP_CREDENTIALS_FILE_PATH}
```

Comment:
nit: Let's make the ENV var more GCS specific since we also test BigQuery on GCP.